pkgs/top-level/stage.nix

```diff
@@ -257,8 +259,6 @@ let
       crossSystem = {
         isStatic = true;
         parsed = makeMuslParsedPlatform stdenv.hostPlatform.parsed;
-      } // lib.optionalAttrs (stdenv.hostPlatform.system == "powerpc64-linux") {
-        gcc.abi = "elfv2";
```

Comment:
@OPNA2608, if you need a quick fix, the root cause is the deletion of these two lines in 82ff1f5db12f862e226e84254e54a10879f1629b.